### DISCUSSION TITLE
[Question] What IDEs are supported by this plugin?

### DISCUSSION THREAD

**Discussion by CodyMarkix** [reloadware/reloadium#36] [General]
I mainly use VS Code for Python development, and I want to know: is there support for Visual Studio Code?

This software looks like a blessing and I'd like to be able to use it.

**Comment by dkrystki** [comment]
Currently only PyCharm and other JetBrains products are supported.
I'll be working on a plugin for Visual Code soon.

**Reply by rambalachandran** [reply]
It has been over a year, any expected timeline on VSCode plugin?

**Reply by dkrystki** [reply]
VSCode plugin is definitely coming but can't give a reliable timeline at this point. There are a few reasons why. The biggest one is limited bandwidth and amount of work that is involved in maintaining Reloadium core and PyCharm plugin.
I will try to publish a minimal working version at the end of September but can't promise anything.

**Comment by TurBoss** [comment]
hello,

eclipse + pydev here

:D


awesome thanks

**Comment by imbev** [comment]
Request for supporting Vim

**Reply by nyngwang** [reply]
link to my own [comment](https://github.com/reloadware/reloadium/discussions/36#discussioncomment-6023834), which is related.

**Comment by FilipeMarch** [comment]
I would love to see this working on VS Code, because I've never seen an extension that would show the time execution for each line of my program... this would help me to know where I should optimize my code.

**Comment by liudonghua123** [comment]
I also use `vscode` quite a lot.

**Comment by FilipeMarch** [comment]
Any updates?

**Reply by dkrystki** [reply]
Unfortunately still pretty far to be released. I have limited bandwidth at the moment.

**Reply by FilipeMarch** [reply]
> Unfortunately still pretty far to be released. I have limited bandwidth at the moment.

All right, Bro! The best luck on your project! I hope one day to use it in VS Code, it looks super cool!

See ya

**Comment by nyngwang** [comment]
Request for supporting Neovim. Well, this one should be combined into Vim I think.
